src/buildingsInfo.py

```diff
@@ -1,39 +1,40 @@
+    {'buildingId': 'barton-hall-2-floor', 'roomNum': 3695891, 'friendlyNames': ['barton', 'barton dorm', 'barton hall']},
+    {'buildingId': 'bray-hall-basement', 'roomNum': 3695892, 'friendlyNames': ['bray', 'bray hall', 'bray dorm']},
+    {'buildingId': 'brinsmade-1-floor', 'roomNum': 3695893, 'friendlyNames': ['brinsmade', 'brinsmade apartments']},
+    {'buildingId': 'bryckwyck-a-basement', 'roomNum': 3695894, 'friendlyNames': ['bryckwyck a']},
+    {'buildingId': 'bryckwyck-b-basement', 'roomNum': 36958932, 'friendlyNames': ['bryckwyck b']},
+    {'buildingId': 'bryckwyck-c-basement ', 'roomNum': 36958933, 'friendlyNames': ['bryckwyck c']},
+    {'buildingId': 'bryckwyck-d-basement', 'roomNum': 36958934, 'friendlyNames': ['bryckwyck d']},
+    {'buildingId': 'bryckwyck-f-basement', 'roomNum': 36958935, 'friendlyNames': ['bryckwyck f']},
+    {'buildingId': 'bryckwyck-g-basement', 'roomNum': 36958936, 'friendlyNames': ['bryckwyck g']},
+    {'buildingId': 'burdett-barh-b-2fl', 'roomNum': 3695896, 'friendlyNames': ['burdett b', 'barh b']},
+    {'buildingId': 'burdett-barh-c-2-fl', 'roomNum': 3695897, 'friendlyNames': ['burdett c', 'barh c']},
+    {'buildingId': 'burdett-hall-barh-a-fl-1', 'roomNum': 3695895, 'friendlyNames': ['burdett a floor 1', 'barh a floor 1']},
+    {'buildingId': 'burdett-hall-barh-a-fl-2', 'roomNum': 36958945, 'friendlyNames': ['burdett a floor 2', 'barh a floor 2']},
+    {'buildingId': 'burdett-hall-barh-d-fl-1', 'roomNum': 3695898, 'friendlyNames': ['burdett b floor 1', 'barh b floor 1']},
+    {'buildingId': 'burdett-hall-barh-d-fl-2', 'roomNum': 36958946, 'friendlyNames': ['burdett d floor 1', 'barh d floor 2']},
+    {'buildingId': 'cary-hall-basement', 'roomNum': 3695899, 'friendlyNames': ['cary', 'cary hall', 'cary dorm']},
+    {'buildingId': 'colonie-apts-a-1-floor', 'roomNum': 36958910, 'friendlyNames': ['colonie a']},
+    {'buildingId': 'colonie-apts-b-1-floor', 'roomNum': 36958911, 'friendlyNames': ['colonie b']},
+    {'buildingId': 'colonie-apts-c-1-floor', 'roomNum': 36958912, 'friendlyNames': ['colonie c']},
+    {'buildingId': 'colonie-apts-d-1-floor', 'roomNum': 36958913, 'friendlyNames': ['colonie d']},
+    {'buildingId': 'colvin-1-floor', 'roomNum': 36958914, 'friendlyNames': ['colvin', 'colvin apartments']},
+    {'buildingId': 'crockett-hall-basement', 'roomNum': 36958915, 'friendlyNames': ['crockett', 'crockett hall', 'crockett dorm']},
+    {'buildingId': 'davison-hall-basement', 'roomNum': 36958931, 'friendlyNames': ['davison', 'davison hall', 'davison dorm']},
+    {'buildingId': 'e-dorms-basement', 'roomNum': 36958917, 'friendlyNames': ['e', 'e complex']},
+    {'buildingId': 'hall-hall-basement', 'roomNum': 36958918, 'friendlyNames': ['hall', 'hall hall', 'hall squared', 'hall dorm']},
+    {'buildingId': 'hunt-quad-basement', 'roomNum': 36958937, 'friendlyNames': ['hunt', 'hunt hall', 'hunt dorm', 'quad hunt']},
```

Comment:
The one in the basement of Hunt is for all of quad, from what I can remember. So "quad", "quad 1", and "quad 2" at the very least should be added.